packages/teams-js/src/public/sharing.ts

```diff
@@ -10,7 +10,12 @@ import { runtime } from './runtime';
  * For more info, see [Share to Teams from personal app or tab](https://learn.microsoft.com/microsoftteams/platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab)
  */
 export namespace sharing {
+  /** shareWebContent callback function type */
+  export type shareWebContentCallbackFunctionType = (err?: SdkError) => void;
+
+  /** Type of message that can be sent or received by the sharing APIs */
   export const SharingAPIMessages = {
```

Comment:
>export [](http://example.com/codeflow?start=2&length=6)

Should this really be exported and documented? It's only ever used once internally in this file.

Reply:
If we don't export, then all the unit test cases will fail for `sharing.ts`.

Reply:
Should it be marked as internal or hidden in the documentation then?

Reply:
Will add internal tag to it.